server/src/test/java/io/crate/integrationtests/UpdateIntegrationTest.java

```diff
@@ -1132,4 +1127,17 @@ public void test_update_on_table_with_big_refresh_interval_emptied_by_delete_ret
         execute("update test set a = 2 where a = 1");
         assertThat(response).hasRowCount(0L);
     }
+
+    @Test
+    public void test_update_continues_on_error_if_at_least_1_row_was_updated() {
```

Comment:
Is this true? Do we throw an error if happen on the first row?

```suggestion
    public void test_update_continues_on_error() {
```

Reply:
Yep, wrong name, leftovers from the previous attempt to throw an error of no row is updated.